IPython/utils/io.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, stream, fallback=None):
         def clone(meth):
             return not hasattr(self, meth) and not meth.startswith('_')
         for meth in filter(clone, dir(stream)):
-            setattr(self, meth, getattr(stream, meth))
+            setattr(self, meth, getattr(stream, meth, None))
```

Comment:
Would the right thing to do be skipping the `setattr` if no attr is found, rather than setting it as None?

e.g.

```python
try:
    attr = getattr(stream, meth)
except AttributeError:
    pass
else:
    setattr(self, meth, attr)
```